ledfx/effects/twod.py

```diff
@@ -142,10 +141,9 @@ def image_to_pixels(self):
         self.pixels[:copy_length, :] = rgb_array[:copy_length, :]
 
     def log_sec(self):
-        self.start = timeit.default_timer()
```

Comment:
Moving the time update from log_sec to the render method to make it more visible